src/test/regress/expected/multi_mx_reference_table.out

```diff
@@ -909,6 +909,28 @@ LOG:  join order: [ "colocated_table_test" ][ reference join "reference_table_te
 
 SET client_min_messages TO NOTICE;
 SET citus.log_multi_join_order TO FALSE;
--- clean up tables
 \c - - - :master_port
-DROP TABLE reference_table_test, reference_table_test_second, reference_table_test_third;;
+-- issue 3766
```

Comment:
Typically, debug 4 is pretty high,  but as far as I can  see we're good with this example.

Reply:
Yeah the reason I am switching to debug4 only when that is necessary was to prevent flaky logs, it seems that it is working fine.